symengine/catch/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+project(catch)
+
+set(SRC
+    catch.cpp
+)
+
+add_library(catch STATIC ${SRC})
+target_link_libraries(catch symengine)
+
+install(TARGETS catch
+            RUNTIME DESTINATION bin
+            ARCHIVE DESTINATION lib
+            LIBRARY DESTINATION lib
+        )
```

Comment:
I think teuchos should be installed. And its includes changed to
<teuchos/*.h>.

Sent from my mobile phone.
On Jun 26, 2015 2:43 AM, "Sumith" <notifications@github.com> wrote:

> In symengine/catch/CMakeLists.txt
> <https://github.com/sympy/symengine/pull/484#discussion_r33338198>:
>
> > @@ -0,0 +1,14 @@
> > +project(catch)
> > +
> > +set(SRC
> > +    catch.cpp
> > +)
> > +
> > +add_library(catch STATIC ${SRC})
> > +target_link_libraries(catch symengine)
> > +
> > +install(TARGETS catch
> > +            RUNTIME DESTINATION bin
> > +            ARCHIVE DESTINATION lib
> > +            LIBRARY DESTINATION lib
> > +        )
>
> So can these 4 lines can be removed?
> I just followed Teuchos on T. Does Teuchos needs to be installed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/symengine/pull/484/files#r33338198>.
>